ui/app/js/controllers/user_edit_controller.js

```diff
@@ -148,7 +152,7 @@ function ($scope, $modalInstance, CSRF, Permissions, user, users) {
       })
       .error(function(response) {
         if (typeof response === 'object') {
-          $scope.errors = response;
+          $scope.errors.permission = response;
```

Comment:
There can be cases I suppose. If we enter an incorrect "permission" and "options" then the error message displayed will be "No JSON object decoded".
But if we re-enter the same permission for a given username it will only be displaying the sweetalert probably rather than highlighting the field.